static_test_driver/static_test_driver.ino

```diff
@@ -235,40 +234,40 @@ void loop() {
   
   BEGIN_READ
   READ_FLAG(zero_force) {
-    #if CONFIGURATION == MK_2
     Serial.println(F("Zeroing load cell"));
     scale.tare(); // Load Cell, Assuming there is no weight on the scale, reset to 0
-
-    #else
-    if (force_zero_ready) {
-      Serial.println(F("Zeroing force plate"));
-      force_zero_ready = false;
-      force_zero_val = 0;
-      for (int i = 0; i < FORCE_PLATE_NUM_HIST_VALS; i++)
-        force_zero_val += force_hist_vals[i];
-      force_zero_val /= FORCE_PLATE_NUM_HIST_VALS;
+  }
+  
+  //TODO: Update flag names to match the following code
+  READ_FLAG(zero_pressure_1) {
+    if (pressure_zero_ready_1) {
+      Serial.println(F("Zeroing fuel pressure"));
+      pressure_zero_ready_1 = false;
+      pressure_zero_val_1 = 0;
```

Comment:
Since this code is now being duplicated you can probably factor it out into a function, at least partially.  Maybe write a function that computes the mean value of an array of floats and use that here instead?